test/tests/functional/pbs_verify_log_output.py

```diff
@@ -113,31 +112,30 @@ def test_if_info_add(self):
         """
         Test for interface info presence in log files
         """
-        names = "["
-        for name in [x[0] for x in self.all_interfaces()]:
-            names += "(" + name + ")"
-        names += "]"
+        interfaceGenerator = self.all_interfaces()
+        for ifname in interfaceGenerator:
+            name = "[(" + ifname + ")]"
 
-        log_val = "[(ipv4)(ipv6)(ipv4/ipv6)] [(interface: )]" + names
+            log_val = "[( interface: )]" + name
```

Comment:
I'm a little confused on your regex here.  Could you explain it to me?  From my understanding, the []s mean match one letter out of the set.  The ()s mean match the regex inside the () and make it a named group you can refer to later.  I don't see how these two work together.  I also don't know why ()s would be useful if you're just going to pass it to log_match().  It's not going to refer to the named groups.